### PR TITLE
SRv6 Test Cases on 7 node testbed Infra changes part

### PR DESCRIPTION
**Pull request by eddieruan-alibaba**
### Description of PR

This part contains infra changes only for enabling 7 node testbed's traffic test cases.

The test case would be added in [another PR.](https://github.com/sonic-net/sonic-mgmt/pull/15723)

Summary:
Fixes # (issue) N/A

### Type of change

- [ ] Bug fix
- [ X] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [ ] 202405

#### What is the motivation for this PR?
Need to run ptf traffic test on 7 node testbed.

#### How did you do it?
Add 5 more test cases, including ptf traffic test and link flapping test cases. 

#### How did you verify/test it?
Via daily jenkins run

#### Any platform specific information?
Only on 7 node vsonic testbed.

### Documentation

https://github.com/sonic-net/sonic-mgmt/blob/master/docs/testplan/srv6/SRv6-phoenixwing-ptf-testplan.md

### CODE REVIEW

File: ansible/roles/vm_set/library/vm_topology.py [hunk 1]
```diff
@@ -1153,8 +1161,8 @@ def bind_ovs_ports(self, br_name, dut_iface, injected_iface, vm_iface, disconnec
                            (br_name, dut_iface_id, vm_iface_id, injected_iface_id))
             VMTopology.cmd("ovs-ofctl add-flow %s table=0,priority=5,ip,in_port=%s,action=output:%s" %
                            (br_name, dut_iface_id, injected_iface_id))
-            VMTopology.cmd("ovs-ofctl add-flow %s table=0,priority=5,ipv6,in_port=%s,action=output:%s" %
-                           (br_name, dut_iface_id, injected_iface_id))
+            VMTopology.cmd("ovs-ofctl add-flow %s table=0,priority=5,ipv6,in_port=%s,action=output:%s,%s" %
+                           (br_name, dut_iface_id, vm_iface_id, injected_iface_id))
```
Comment (wangxin) [inline ansible/roles/vm_set/library/vm_topology.py:1165]: Can you submit a separate PR for this infrastructure code change made to "vm_topology.py" file?

Comment (eddieruan-alibaba) [reply ansible/roles/vm_set/library/vm_topology.py:1165]: These changes are required for enabling PTF traffic to go through 7 node testbed. Current vm topology only assumes one direction of the traffic from other node to DUT. For 7 nodes testbed, we need to have traffic running from DUT and to DUT. That's why we need to make these changes.

Comment (wangxin) [reply ansible/roles/vm_set/library/vm_topology.py:1165]: Is it a better idea to separate infrastructure change and test case change in different PRs?

**Review by wangxin — APPROVED** [review]

File: tests/common/plugins/ptfadapter/__init__.py [hunk 2]
```diff
@@ -77,14 +78,25 @@ def get_ifaces_map(ifaces, ptf_port_mapping_mode):
...
         else:
             iface_index = int(iface_suffix)
             iface_map[iface_index] = iface
+            used_index.add(iface_index)
+
+    count = 1
+    while count in used_index:
+        count = count + 1
+    if backplane_exist:
+        iface_map[count] = "backplane"
```
Comment (lizhijianrd) [inline tests/common/plugins/ptfadapter/__init__.py:99]: Hi @eddieruan-alibaba, could you please help explain why `backplane` is added to `iface_map`?
